frontend/public/components/droppable-edit-yaml.tsx

```diff
@@ -42,10 +52,17 @@ export const DroppableEditYAML = withDragDropContext(class DroppableEditYAML ext
     if (file.size <= maxFileUploadSize) {
       const reader = new FileReader();
       reader.onload = () => {
-        const input = reader.result;
-        this.setState({
-          fileUpload: input,
-        });
+        const input:string = reader.result as string;
```

Comment:
Is TypeScript preventing you from assigning this to a string without `as string`? What's the compiler error?

Reply:
@spadgett yes, its ```
S2322: Type 'string | ArrayBuffer' is not assignable to type 'string'.
  Type 'ArrayBuffer' is not assignable to type 'string'.
```

Reply:
OK, I believe we know it's always a string in this case since we call `readAsText`. You don't need to `: string` if you use `as string`. This should be safe:

```suggestion
        const input = reader.result as string;
```

Reply:
@spadgett ok, all set.